### PR TITLE
fix 'bad matcher format: Watchdog' and some post merge cleanup

### PR DESCRIPTION
**Pull request by pixelsoccupied**
/cc @mlguerrero12 
/cc @alegacy

### CODE REVIEW

File: internal/service/alarms/internal/alertmanager/alertmanager.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ route:
   - receiver: "null"
     matchers:
       # Always firing alert to verify alertmanager is working
-      - Watchdog
+      - alertname="Watchdog"
```
Comment (pixelsoccupied) [inline internal/service/alarms/internal/alertmanager/alertmanager.yaml:15]: Wonder if we should Unmarshal it to Go struct to possible check for any config error validation.

Comment (mlguerrero12) [inline internal/service/alarms/internal/alertmanager/alertmanager.yaml:15]: I fixed but didn't push the commit. Thanks! Quotes are not needed. Only in for the null receiver

File: internal/service/alarms/serve.go [hunk 2]
```diff
@@ -13,11 +13,12 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/openshift-kni/oran-o2ims/internal/service/alarms/internal/alertmanager"
+
```
Comment (mlguerrero12) [inline internal/service/alarms/serve.go:17]: let's put it together with the other imports

Comment (pixelsoccupied) [reply internal/service/alarms/serve.go:17]: I don't modify it manually actually. Simply run `golangci-lint run --fix --config .golangci.yml` and let it do its thing.

File: internal/service/alarms/internal/alertmanager/alertmanager.yaml (hunk 1 above)
Comment (pixelsoccupied) [reply internal/service/alarms/internal/alertmanager/alertmanager.yaml]: Updated! Wonder if we should Unmarshal it to Go struct to possibly validate any error from plain text file

File: internal/service/alarms/serve.go (hunk 2 above)
Comment (mlguerrero12) [reply internal/service/alarms/serve.go:17]: weird, we've been grouping this kind of imports together. Lint locally and CI do not complain.

File: internal/service/alarms/internal/alertmanager/alertmanager.yaml (hunk 1 above)
Comment (mlguerrero12) [reply internal/service/alarms/internal/alertmanager/alertmanager.yaml]: I think once we have integration tests we could detect this kind of issues. Besides, inside the matchers, you can put whatever you want.

File: internal/service/alarms/serve.go (hunk 2 above)
Comment (pixelsoccupied) [reply internal/service/alarms/serve.go]: updated manually

**Pull request merged**